### DISCUSSION TITLE
FindMySync on sequoia

### DISCUSSION THREAD

**Discussion by dssenthil** [MartinPham/FindMySync#39] [General]
Noticed that FindMySync stopped working on sequoia. Any one have workaround for this? Thanks in advance. 

`MacBook-Pro:~ $ security find-generic-password -l BeaconStore -g
security: SecKeychainSearchCopyNext: The specified item could not be found in the keychain.
MacBook-Pro:~$`

<img width="1214" alt="Screenshot 2024-09-27 at 1 42 37 PM" src="https://github.com/user-attachments/assets/e1133713-35b1-4fd1-a988-70c1c020c263">
<img width="1265" alt="Screenshot 2024-09-27 at 1 38 43 PM" src="https://github.com/user-attachments/assets/0aa4b01d-726b-48bf-8bbd-738449049480">

**Comment by wilkyconsultants** [comment]
What MacOS version were you using before you went to sequoia where the security command was working for you? The reason I ask is I am using Sonoma 14.7 and I had to run the following to get the output My issue is when I use the password to try and decrypt the ~/Library/Caches/com.apple.findmy.fmipcore/Items.list file it appears it is the wrong key to use as I get "Decryption failed: MAC check failed" which means wrong key Hmm.. Wonder where Apple stuffed that key?

security unlock-keychain -p -Removed- /Users/tracker/Library/Keychains/login.keychain-db
security find-generic-password -l "BeaconStore" -g  

keychain: "/Users/tracker/Library/Keychains/login.keychain-db"
version: 512
class: "genp"
attributes:
    0x00000007 <blob>="BeaconStore"
    0x00000008 <blob>=<NULL>
    "acct"<blob>="BeaconStoreKey"
    "cdat"<timedate>=0x32303235303232333230303532315A00  "20250223200521Z\000"
    "crtr"<uint32>=<NULL>
    "cusi"<sint32>=<NULL>
    "desc"<blob>=<NULL>
    "gena"<blob>=<NULL>
    "icmt"<blob>=<NULL>
    "invi"<sint32>=<NULL>
    "mdat"<timedate>=0x32303235303232333232343933355A00  "20250223224935Z\000"
    "nega"<sint32>=<NULL>
    "prot"<blob>=<NULL>
    "scrp"<sint32>=<NULL>
    "svce"<blob>="BeaconStore"
    "type"<uint32>=<NULL>
password: 0x-Removed- ,"
